rustfmt-core/rustfmt-config/src/lib.rs

```diff
@@ -169,6 +169,7 @@ impl PartialConfig {
         cloned.width_heuristics = None;
         cloned.print_misformatted_file_names = None;
         cloned.recursive = None;
+        cloned.emit_mode = None;
 
```

Comment:
I think `skip_children` up on line 130 would need to be removed too to fix the failing `test_dump_default_config` test right?